WebContent/js/containers/ContentViewer.js

```diff
@@ -92,7 +100,22 @@ export class ContentViewer extends React.Component {
         dispatch(removeContent(removeIndex));
         // Do we need to change the selectedContent
         if (removeIndex <= selectedContent && selectedContent >= 1) {
-            dispatch(changeSelectedContent(selectedContent - 1));
+            const newTabIndex = selectedContent - 1;
+            dispatch(changeSelectedContent(newTabIndex));
+        }
+    }
+
+    handleKeyDown(e, index) {
+        if (e.key === 'Enter') { this.handleSelectedTabChange(index); }
+        // if (e.altKey && e.keyCode === 87) { this.handleCloseTab(index); }
+    }
+
+    focusToActiveTab() {
+        const { selectedContent } = this.props;
+        const tab = this.fileTabs[selectedContent];
+        if (tab) {
+            tab.focus();
+            // tab.scrollIntoView({ behavior: 'smooth', inline: 'center', block: 'center' });
```

Comment:
We shouldn't include commented out code which is unused, it will go stale

Reply:
was trying not to use focus, but other way for some reason was not working here. removed